README.md

```diff
@@ -40,10 +40,10 @@ To add a dependency using Maven:
     * [Scalar DB: A library that makes non-ACID databases ACID-compliant](https://www.slideshare.net/scalar-inc/scalar-db-a-library-that-makes-nonacid-databases-acidcompliant) at Database Lounge Tokyo #6 2020
     * [Transaction Management on Cassandra](https://www.slideshare.net/scalar-inc/transaction-management-on-cassandra) at Next Generation Cassandra Conference / ApacheCon NA 2019
 * Javadoc
-    * [scalardb](https://javadoc.io/doc/com.scalar-labs/scalardb/latest/index.html) - A library that makes non-ACID distributed databases/storages ACID-compliant
+    * [scalardb](https://javadoc.io/doc/com.scalar-labs/scalardb/latest/index.html) - Scalar DB: A universal transaction manager that achieves database-agnostic transactions and transactions that span multiple databases
```

Comment:
```suggestion
    * [scalardb](https://javadoc.io/doc/com.scalar-labs/scalardb/latest/index.html) - Scalar DB: A universal transaction manager that achieves database-agnostic transactions and distributed transactions that span multiple databases
```
Sorry, I wrote the above description but I realized adding `distributed` sounds more appropriate and natural.

Reply:
Fixed in 94c298261a08a156800a16c3a1781584b2f9b168. Please take a look again when you have time!